src/game.zig

```diff
@@ -401,6 +402,12 @@ pub const Player = struct { // MARK: Player
 		return eyeCoyote;
 	}
 
+	pub fn getFovModifierBlocking() f32 {
+		mutex.lock();
+		defer mutex.unlock();
+		return fovMod;
+	}
+
```

Comment:
Currently not used, but it would be if the renderer asks the player what the FOV should be